src/Uno.UWP/Storage/FileProperties/BasicProperties.cs

```diff
@@ -7,16 +7,24 @@ namespace Windows.Storage.FileProperties
 {
 	public sealed partial class BasicProperties
 	{
-		private readonly FileInfo _fileInfo;
+		internal BasicProperties(ulong size, DateTimeOffset dateModified)
```

Comment:
This means that those properties are now immutable. Even if I like it, I'm wondering if it's the case on UWP?

Reply:
Yes, the `BasicProperties` are read-only and when the file is modified, the changes are not reflected there (unless `GetBasicPropertiesAsync` is called again)